internal/model/events.go

```diff
@@ -0,0 +1,8 @@
+package model
+
+// EventMetaDefault is a standard embedded struct to be used with custom event metadata types, gives an interface for
```

Comment:
I wonder how we misconfigured the linter this time. This isn't valid godoc, missing trailing `.`